doc/source/walkthrough.rst

```diff
@@ -26,6 +26,8 @@ Installation
 
     To run this walkthrough, add `Ray API <https://mvnrepository.com/artifact/io.ray/ray-api>`_ and `Ray Runtime <https://mvnrepository.com/artifact/io.ray/ray-runtime>`_ as dependencies. Snapshot versions can be found in `sonatype repository <https://oss.sonatype.org/#nexus-search;quick~io.ray>`_.
 
+    Note: To run your Ray Java application in cluster mode, you must install it by `pip install -U ray` first. To run it in single process mode, there is no need to do this installation.
```

Comment:
```suggestion
    Note: To run your Ray Java application, you need to install Ray Python with `pip install -U ray` first. (For Ray Java snapshot versions, install nightly Ray Python wheels.) The versions of Ray Java and Ray Python must match.
```

I think we don't need to mention local mode (in the document we call it local mode with the configuration `ray.local-mode=true`) here as it may distract users when they want to try Ray Java for the first time. We also don't recommend local mode except for debugging purposes.